source/plugins/ruby/in_win_cadvisor_perf.rb

```diff
@@ -70,6 +69,7 @@ def enumerate()
           @@winNodeQueryTimeTracker = DateTime.now.to_time.to_i
         end
         @@winNodes.each do |winNode|
+          eventStream = MultiEventStream.new
```

Comment:
the insightsmetrics one is also adding (for GPU). Currently gpu sku is not supported on windows, but good to new that on every iteration as well :)

Reply:
oh yeah. i missed that. i will fix it.

Reply:
Yes I think the PV metrics duplicate too. I had tested the PV changes with only one windows node...so didn't notice that this would happen. Good to know I should always test with multiple 👍

Reply:
@gracewehner - i think this should fix the PV metrics too right?? Did i miss something else?

Reply:
No you're good, moving the insightsmetricseventstream = MultiEventStream.new will fix it, just wanted to clarify :)

Reply:
okay.. no worries 👍